rasa/train.py

```diff
@@ -674,19 +675,20 @@ async def _train_nlu_with_validated_data(
         return _train_path
 
 
-def _nlu_model_for_finetuning(model_to_finetune: Text) -> Optional[Interpreter]:
-    from rasa.core.interpreter import RasaNLUInterpreter
-
+def _nlu_model_for_finetuning(
+    model_to_finetune: Text,
+    new_config: Dict[Text, Any],
+    finetuning_epoch_fraction: float = 1.0,
+) -> Optional[Interpreter]:
     path_to_archive = model.get_model_for_finetuning(model_to_finetune)
     if not path_to_archive:
         return None
 
-    try:
```

Comment:
I thought it's might be a good idea to just train from scratch in case anything goes wrong. On the other hand this can hide issues. If we remove it, we should do so, too, for the Core model loading.

Reply:
I would say better to fail fast and explicitly - especially as it's an experimental feature.

Reply:
Good point!